.github/workflows/docker.yml

```
@@ -0,0 +1,51 @@
+name: Build and publish Docker image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build Docker image
+    runs-on: ubuntu-18.04
```

```suggestion
    runs-on: ubuntu-22.04
```

I'd recommend using a later Ubuntu version than 18.04.